accounts/abi/bind/bind_test.go

```diff
@@ -1708,7 +1716,7 @@ func TestGolangBindings(t *testing.T) {
 	defer os.RemoveAll(ws)
 
 	pkg := filepath.Join(ws, "bindtest")
-	if err = os.MkdirAll(pkg, 0700); err != nil {
+	if err = os.MkdirAll(pkg, 0o700); err != nil {
```

Comment:
Do you know difference between `0700` vs `0o700`?

Reply:
The same. The prefix `0`, `0o`, and `0O` indicate an octal integer literal. Check this link https://go.dev/doc/go1.13